src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -1555,23 +1571,23 @@ void program_node::init_onednn_primitive_attributes() {
                             post_ops.append_eltwise(dnnl::algorithm::eltwise_linear, q_param->_in_scale, 0.0f);
                             update_onednn_post_op_list(onednn_post_op_type::eltwise_linear, empty_mem);
                         } else {
-                            auto in_scale = get_dependency(dep_idx++).get_output_layout();
+                            auto in_scale = get_input_layout(dep_idx++);
                             dnnl::memory::desc in_scale_desc = onednn::layout_to_memory_desc(in_scale, dnnl::memory::format_tag::ab, true);
                             post_ops.append_binary(dnnl::algorithm::binary_mul, in_scale_desc);
                             update_onednn_post_op_list(onednn_post_op_type::binary_mul, dep_idx - 1, dnnl::memory::format_tag::ab, true,
-                                                       in_scale_desc.get_dims(), in_scale_desc.get_data_type());
+                                                        in_scale_desc.get_dims(), in_scale_desc.get_data_type());
```

Comment:
No need to change it. Please update it too?

Reply:
I do not modify it manually but I rollback it.